app/src/main/java/com/framgia/forder/screen/updateProduct/UpdateProductViewModel.java

```diff
@@ -23,10 +23,12 @@
 import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
 
 import static com.framgia.forder.screen.createProduct.CreateProductFragment.REQUEST_SELECT_IMAGE;
 import static com.framgia.forder.utils.Constant.FLAG_END_HOUR;
 import static com.framgia.forder.utils.Constant.FLAG_OPEN_HOUR;
+import static com.framgia.forder.utils.Constant.FORMAT_PRICE;
```

Comment:
ERROR: Unused import: com.framgia.forder.utils.Constant.FORMAT_PRICE. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports